### ISSUE TITLE
Accessibility: replace screenshot of table with HTML table

### ISSUE THREAD

**Issue by tobyhodges** [datacarpentry/spreadsheet-ecology-lesson#319]
The table evaluating different choices of null values in [the _Using problematic null values_ section of the _Formatting Problems_ episode](https://datacarpentry.org/spreadsheet-ecology-lesson/02-common-mistakes/index.html#-using-problematic-null-values) is currently a screenshot from the PDF of [the paper it was originally published in](https://ojs.library.queensu.ca/index.php/IEE/article/view/4608). 

This format is not accessible to people accessing the lesson with a screen reader, and it is very difficult to capture all the information in the table as alt-text to the image. I believe the screenshot should be replaced with an HTML table containing the same information.

Note that there is [an identical issue in the _Data Organization in Spreadsheets for Social Scientists_ lesson repository](https://github.com/datacarpentry/spreadsheets-socialsci/issues/151), and another in [the Library Carpentry _Tidy Data for Librarians_ lesson repository](https://github.com/LibraryCarpentry/lc-spreadsheets/issues/124), which should be addressed simultaneously. The Library Carpentry repository issue also includes some suggestions for how the descriptions in the table could be improved when converting the screenshot.

**Comment by ethanwhite** [comment]
The table is available in markdown in the original repo here: https://github.com/weecology/data-sharing-paper/blob/master/data_sharing_ms.md. Though tables in md were a bit of a mess a decade ago so it may need some work.

**Comment by hoytpr** [comment]
@tobyhodges @ethanwhite The .md table looks fine on my end using the Markdown Panel fro Notepad++.  If it renders well on my local copy of the repo, I'll setup a PR. (Might take a while I've been out-of-service and am behind in maintenance)

**Comment by hoytpr** [comment]
@ethanwhite @tobyhodges PR submitted

**Comment by hoytpr** [comment]
@tobyhodges I fixed the second table and accidentally merged it into the main site. GitHub hates me. But it looks okay.

**Comment by hoytpr** [comment]
> Thanks for this material, I am working on incorporating into the Library Carpentry version. I like what's being proposed here, but it recreates the table from the original article, which is not ideal. The table is still difficult to understand, as noted at one of the discussions that raised this (LC discussion at [LibraryCarpentry/lc-spreadsheets#124](https://github.com/LibraryCarpentry/lc-spreadsheets/issues/124)).
> 
> @tobyhodges Has it been considered reordering the rows in this table? Would it be possible to put the suggested values either at the top or bottom, and possibly highlight them some way?

Yes @tobyhodges already requested we look at the suggestions in https://github.com/LibraryCarpentry/lc-spreadsheets/issues/124) and work together to implement improvements. This was a partial goal... to get the table into HTML. Should we head over to https://github.com/LibraryCarpentry/lc-spreadsheets/issues/124 and work on this over there? There appear to be more maintainers actively involved.

**Comment by hoytpr** [comment]
Closing this. Please open new issue for any rewriting of the table.
